src/app/shared/view-mode-switch/view-mode-switch.component.html

```diff
@@ -40,5 +40,20 @@
       [attr.data-test]="'detail-view' | dsBrowserOnly">
       <i class="far fa-square"></i>
     </button>
+    @if (isToShow(viewModeEnum.GeospatialMap)) {
+      <button
+        [attr.aria-current]="currentMode === viewModeEnum.GeospatialMap"
+        [attr.aria-label]="'search.view-switch.show-geospatialMap' | translate"
+         routerLink="."
+         [queryParams]="{view: 'geospatialMap'}"
+         queryParamsHandling="merge"
+         (click)="switchViewTo(viewModeEnum.GeospatialMap)"
+         routerLinkActive="active"
+         [class.active]="currentMode === viewModeEnum.GeospatialMap"
+         class="btn btn-secondary"
+         [attr.data-test]="'geospatial-view' | dsBrowserOnly">
+         <span class="fas fa-map"></span><span class="sr-only">{{'search.view-switch.show-geospatialMap' | translate}}</span>
+      </button>
+    }
```

Comment:
The map icon is not being displayed. The condition is inside "`@if (isToShow(viewModeEnum.DetailedListElement)) {`", but it should be after the` }` on line 58.